charts/logging-pipeline-plumber/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-name: rancher-logging-explorer
+name: logging-pipeline-plumber
```

Comment:
if we do this we need to make sure we are consistent everywhere including the repository name

Reply:
I think the new name is more descriptive of what the project does though

Reply:
yeah it's updated everywhere with this PR, except repo name in GitHub